src/integrate/orchestrator.rs

```diff
@@ -0,0 +1,1204 @@
+                ),
+            ]),
+            host_config: Some(HostConfig {
+                port_bindings: Some(port_bindings),
+                ..Default::default()
+            }),
+            ..Default::default()
+        };
+
+        let options = CreateContainerOptions {
+            name: format!("bitcoin-explorer.{}", self.network_name),
+        };
+
+        let container = docker
+            .create_container::<String, String>(Some(options), config)
+            .await
+            .map_err(|e| format!("Unable to create container: {}", e))?
+            .id;
+
+        self.bitcoin_explorer_container_id = Some(container.clone());
+
+        docker
+            .start_container::<String>(&container, None)
+            .await
+            .map_err(|e| format!("Unable to create container: {}", e))?;
+
+        let res = docker
+            .connect_network(
+                &self.network_name,
+                ConnectNetworkOptions {
+                    container,
+                    ..Default::default()
+                },
+            )
+            .await;
+
+        if let Err(e) = res {
+            let err = format!("Error connecting container: {}", e);
+            println!("{}", err);
+            return Err(err);
+        }
+
+        Ok(())
+    }
+
+    pub async fn restart(&mut self) {}
+
+    pub async fn terminate(&mut self) {
+        let (docker, devnet_config) = match (&self.docker_client, &self.network_config) {
+            (Some(ref docker), Some(ref network_config)) => match network_config.devnet {
+                Some(ref devnet_config) => (docker, devnet_config),
+                _ => return,
+            },
+            _ => return,
+        };
+
+        println!("Stopping Devnet...");
+
+        let options = Some(KillContainerOptions { signal: "SIGKILL" });
+
+        // Terminate containers
+        if let Some(ref bitcoin_explorer_container_id) = self.bitcoin_explorer_container_id {
+            println!("Terminating bitcoin-explorer...");
+            let _ = docker
+                .kill_container(bitcoin_explorer_container_id, options.clone())
+                .await;
+            let _ = docker.remove_container(bitcoin_explorer_container_id, None);
+        }
+    
+        if let Some(ref stacks_explorer_container_id) = self.stacks_explorer_container_id {
+            println!("Terminating stacks-explorer...");
+            let _ = docker
+                .kill_container(stacks_explorer_container_id, options.clone())
+                .await;
+        }
+
+
+        if let Some(ref bitcoin_blockchain_container_id) = self.bitcoin_blockchain_container_id {
+            println!("Terminating bitcoin-node...");
+            let _ = docker
+                .kill_container(bitcoin_blockchain_container_id, options.clone())
+                .await;
+            let _ = docker.remove_container(bitcoin_blockchain_container_id, None);
+        }
```

Comment:
We should probably terminate this container last since it's a dependency of the `stacks-node` container, inversely of how it's the first container to be started. Doing it in this current order _shouldn't_ hurt anything, but it's a good practice to follow in general.

Reply:
In this we're not trying to gracefully terminate services, just killing everything, so order doesn't really matter